test/modules/linear_boltzmann_solvers/transport_keigen/c5g7/c5g7_jfnk.lua

```diff
@@ -0,0 +1,98 @@
+--- Final k-eigenvalue    :         1.1925596 (265)
+
+dofile("mesh/gmesh_coarse.lua")
+dofile("materials/materials.lua")
+
+-- Setup Physics
+
+-- Angular quadrature
+pquad = aquad.CreateProductQuadrature(GAUSS_LEGENDRE_CHEBYSHEV,2, 2)
+aquad.OptimizeForPolarSymmetry(pquad, 4.0*math.pi)
+
+-- Solver
+phys1 = lbs.DiscreteOrdinatesSolver.Create
+({
+    num_groups = num_groups,
+    groupsets =
+    {
+        {
+            groups_from_to = {0, num_groups-1},
+            angular_quadrature_handle = pquad,
+            inner_linear_method = "gmres",
+            l_max_its = 5,
+            l_abs_tol = 1.0e-10,
+            angle_aggregation_type = "polar",
+            angle_aggregation_num_subsets = 1,
+            groupset_num_subsets = 1,
+        }
+    },
+    options =
+    {
+        boundary_conditions = { { name = "xmax", type = "reflecting"},
+                                { name = "ymax", type = "reflecting"} },
+        scattering_order = 1,
+        verbose_outer_iterations = false,
+        verbose_inner_iterations = false,
+        power_field_function_on = true,
+        power_default_kappa = 1.0,
+        power_normalization = 1.0,
+        save_angular_flux = true
+    },
+    sweep_type = "AAH"
+})
+
+-- Execute Solver
+if (k_method == "pi") then
+    k_solver = lbs.XXPowerIterationKEigen.Create
+    ({
```

Comment:
still XX in name?

Reply:
Yes, the other PR hasn't been merged yet. When it is, I'll rebase and update this one.